.circleci/config.yml

```diff
@@ -24,206 +24,244 @@ commands:
+    - cypress_tests_linux:
         name: "UI Tests - Chrome"
         browser: chrome
-        spec: cypress/tests/ui/*
-        executor: with-chrome-and-firefox
-        wait-on: "http://localhost:3000"
-        command-prefix: npx percy exec --
-        yarn: true
-        start: yarn start:ci
-        record: true
-        parallel: true
-        parallelism: 5
-        ci-build-id: ${CIRCLE_SHA1:0:8}
+        specPattern: "cypress/tests/ui/*"
+        ciBuildId: ${CIRCLE_SHA1:0:8}
         group: "UI - Chrome"
         requires:
           - Setup Linux
-        post-steps:
-          - report-coverage
-
     # Run E2E tests in Chrome with mobile device viewport
-    - cypress/run:
+    - cypress_tests_linux:
         name: "UI Tests - Chrome - Mobile"
         browser: chrome
-        spec: cypress/tests/ui/*
-        config: "viewportWidth=375,viewportHeight=667"
-        executor: with-chrome-and-firefox
-        wait-on: "http://localhost:3000"
-        yarn: true
-        start: yarn start:ci
-        record: true
-        parallel: true
-        parallelism: 5
-        ci-build-id: ${CIRCLE_SHA1:0:8}
+        specPattern: "cypress/tests/ui/*"
+        ciBuildId: ${CIRCLE_SHA1:0:8}
         group: "UI - Chrome - Mobile"
+        isMobile: true
         requires:
           - Setup Linux
-        post-steps:
-          - report-coverage
-
     # Run E2E tests in Firefox
-    - cypress/run:
+    - cypress_tests_linux:
         name: "UI Tests - Firefox"
         browser: firefox
-        spec: cypress/tests/ui/*
-        executor: with-chrome-and-firefox
-        wait-on: "http://localhost:3000"
-        yarn: true
-        start: yarn start:ci
-        record: true
-        parallel: true
-        parallelism: 5
-        ci-build-id: ${CIRCLE_SHA1:0:8}
+        specPattern: "cypress/tests/ui/*"
+        ciBuildId: ${CIRCLE_SHA1:0:8}
         group: "UI - Firefox"
         requires:
           - Setup Linux
-        post-steps:
-          - report-coverage
-
     # Run E2E tests in Firefox with mobile device viewport
-    - cypress/run:
+    - cypress_tests_linux:
         name: "UI Tests - Firefox - Mobile"
         browser: firefox
-        spec: cypress/tests/ui/*
-        config: viewportWidth=375,viewportHeight=667
-        executor: with-chrome-and-firefox
-        wait-on: "http://localhost:3000"
-        yarn: true
-        start: yarn start:ci
-        record: true
-        parallel: true
-        parallelism: 5
-        ci-build-id: ${CIRCLE_SHA1:0:8}
+        specPattern: "cypress/tests/ui/*"
+        ciBuildId: ${CIRCLE_SHA1:0:8}
         group: "UI - Firefox - Mobile"
+        isMobile: true
         requires:
           - Setup Linux
-        post-steps:
-          - report-coverage
 
 windows-workflow: &windows-workflow
   jobs:
-    # Setup
-    #  1. Install Cypress
-    #  2. Validate types
-    #  3. Run server unit-tests
-    - cypress/install:
+    - setup_project_and_cypress_windows:
         name: "Setup Windows"
```

Comment:
is this doing anything?

Reply:
oh I see this is reusing a job

Reply:
the job or the name?

Reply:
name is just there to keep parity between the update so it doesn't look different in circle